ivi-layermanagement-api/ilmControl/src/ilm_control_wayland_platform.c

```diff
@@ -193,11 +193,33 @@ output_listener_scale(void *data,
     (void)factor;
 }
 
+static void
+output_listener_name(void *data,
+                      struct wl_output *output,
+                      const char *name)
+{
+    (void)data;
+    (void)output;
+    (void)name;
+}
+
+static void
+output_listener_description(void *data,
+                      struct wl_output *output,
+                      const char *description)
+{
+    (void)data;
+    (void)output;
+    (void)description;
+}
+
 static struct wl_output_listener output_listener = {
     output_listener_geometry,
     output_listener_mode,
     output_listener_done,
-    output_listener_scale
+    output_listener_scale,
```

Comment:
the new added events are part of the wl_output protocol version 3 and above but we are registering the protocol version 1 in https://github.com/COVESA/wayland-ivi-extension/blob/a0529d51134c4d0b3b660be815aea194ff8cc24d/ivi-layermanagement-api/ilmControl/src/ilm_control_wayland_platform.c#L957

please align this otherwise the callback will never be triggered, we should in general consider if we really need thos new events in the wayland-ivi-extentions and we should also think if we need to support several wl_output protocol versions (my current view on this we should not, because we can run only on weston and we need to ensure the compartible weston version has the corresponding support for the protocol version we are using) mostly it would make sense just to support same protocol version as supported in the corresponding weston version.

same is true for other protocols in this commit. please check this

Reply:
I have one confuse. According to the definition in this xml file [(link)](https://gitlab.freedesktop.org/wayland/wayland/-/blob/main/protocol/wayland.xml?ref_type=heads#L2818), the "done" and "scale" events are part of the wl_output protocol version 2 and above. But for now we are registering the protocol version 1[(link)](https://github.com/COVESA/wayland-ivi-extension/blob/a0529d51134c4d0b3b660be815aea194ff8cc24d/ivi-layermanagement-api/ilmControl/src/ilm_control_wayland_platform.c#L957). So listeners of these events will never be triggered, but they still implemented in here. Do you know why?